HOWTO/INSTALL.md

```diff
@@ -549,48 +549,54 @@ If you have Xcode 4.3, or later, you will also need to download
 
 #### Building with wxErlang ####
 
-If you want to build the `wx` application, you will need to get wxWidgets-3.0
-(`wxWidgets-3.0.3.tar.bz2` from <https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.3/wxWidgets-3.0.3.tar.bz2>) or get it from github with bug fixes:
+wxWidgets-3.2.x is required for building the `wx` application.
+Download it from <https://www.wxwidgets.org/downloads> or from
+<https://github.com/wxWidgets/wxWidgets>. It is recommended to use the
+latest release in the 3.2 series, which at the time of writing
+is 3.2.2.1.
 
-    $ git clone --branch WX_3_0_BRANCH git@github.com:wxWidgets/wxWidgets.git
+Note that the wxWidgets-3.3 versions are experimental, but they should
+also work if 3.0 compatibility is enabled by adding
+`--enable-compat30` to the `configure` commands below.
 
-The wxWidgets-3.1 version should also work if 2.8 compatibility is enabled,
-add `--enable-compat28` to configure commands below.
+For Windows, it is easiest to download pre-built binaries.
```

Comment:
Remove the windows comment.

For windows the build instructions are in *HOWTO/INSTALL-WIN32.md* 
can you update the version numbers in that document as well :-) 
replace wxWidgets-3.1.[3|4] to wxWidgets-3.2.